_projects/1_HP_Wizard.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: 1 HP Wizard
+image: /img/projects/1HPWizard.png
+custom-css: project
+desc: |
+  A multi-level, first-person fantasy strategy game
+github: https://github.com/scott0123/1-HP-Wizard
+link: https://github.com/scott0123/1-HP-Wizard/releases
```

Comment:
I'm not sure that that's an appropriate use for `link`, since it links to the github itself.